internal/clients/packer.go

```diff
@@ -39,7 +40,7 @@ func GetPackerChannelBySlug(ctx context.Context, client *Client, loc *sharedmode
 // channel associated with the given channel name, using ListBucketChannels
 func GetPackerChannelBySlugFromList(ctx context.Context, client *Client, loc *sharedmodels.HashicorpCloudLocationLocation,
 	bucketName string, channelName string) (*packermodels.HashicorpCloudPackerChannel, error) {
-	resp, err := ListBucketChannels(ctx, client, loc, bucketName)
+	resp, err := ListPackerChannels(ctx, client, loc, bucketName)
```

Comment:
Trying to align with common pattern where methods/types include the product name, and `PackerBucketChannel` seems redundant/verbose since there is only one kind of Channel in Packer